README.md

```diff
@@ -2,28 +2,73 @@
+run the following:
+```
+pip install tfx-addons
+```
+
+To ensure you have a compatible version of dependencies for any given project, 
+you can specify the project name  as an extra requirement during install:
+
+```
+pip install tfx-addons[feast_examplegen,schema_curation]
+``` 
+
+To use TFX Addons:
+
+```python
+from tfx import v1 as tfx
+import tfx_addons as tfxa
+```
 
-In order to accelerate the sharing of community customizations and additions to TFX, the TFX team would like to encourage, enable, and organize community contributions to help continue to meet the needs of production ML, expand the vision, and help drive new directions for TFX and the ML community.
 
-## Goals & Objectives
-We welcome community contributions on any area of TFX, but this SIG will initially focus on the following goals:
+## Addons Subpackages
 
-- Driving the development of high-quality custom pipeline components, including Python function-based components, container-based components, and fully custom components.
-- Shaping a standardized set of descriptive metadata for community-contributed components to enable easy understanding, comparison, and sharing of components during discovery.
-- Similarly driving the development of templates, libraries, visualizations, and other useful additions to TFX.
+* [tfxa.mlmd_client](tfx_addons/mlmd_client) 
+* [tfxa.schema_curation](tfx_addons/schema_curation) 
+* [tfxa.feast_examplegen](tfx_addons/feast_examplegen) 
+* [tfxa.xgboost_evaluator](tfx_addons/xgboost_evaluator)
+* [tfxa.sampling](tfx_addons/sampling)
+* [tfxa.message_exit_handler](tfx_addons/message_exit_handler) 
 
-These projects will begin as proposals to the SIG, and upon approval will be led and maintained by the community members involved in the project and assigned a project folder, with high-level consultation from the TFX team.
 
-### In-Scope, Out of Scope
-Although TFX is an open-source project and we welcome contributions to TFX itself, **this SIG does not include contributions or additions to core TFX**.  It is focused only on building community-contributed and maintained additions on top of core TFX.  [Core TFX has its own repo](https://github.com/tensorflow/tfx), and PRs and issues will continue to be managed there. **In addition, all contributions must not violate the [Google AI Principles](https://ai.google/principles/) or [Responsible AI Practices](https://ai.google/responsibilities/responsible-ai-practices/).**
 
-## Membership
+## Tutorials and examples
+See [`examples/`](examples/)
+for end-to-end examples of various addons.
+
+## Maintainership
+
+TFX Addons has been designed to compartmentalize submodules so 
+that they can be maintained by community users who have expertise, and a vested 
+interest in that component. We heavily encourage users to submit sign up to maintain a 
+submodule by submitting your username to the [CODEOWNERS](CODEOWNERS) file.
+
+Full write access will only be granted after substantial contribution 
```

Comment:
I think I see where you're going with this, it echos other OSS projects and I agree that it's a good idea.